packages/xo-web/src/common/xoa-updater.js

```diff
@@ -35,6 +35,73 @@ export function blockXoaAccess(xoaState) {
   return block
 }
 
+export function getLicenseNearExpiration(licenses) {
+  if (licenses.find(({ expires }) => expires === undefined) !== undefined) {
```

Comment:
```suggestion
  if (licenses.filter(({ expires }) => expires === undefined).length) {
```
don't know if this solution can work in that case

Reply:
the goal is to not apply any rule if the user has a license without expiration date

Reply:
ok, I let you decide what code is the most relevant then :)